stalkbook/static/angular_code/controllers/app.js

```diff
@@ -21,27 +14,14 @@ module.config(['$routeProvider',
 module.controller("TagController", function($scope, $http) {
 
     $scope.image = ""
-    console.log($scope.image)
     $scope.search = function(image) {
-
-        console.log($scope.image)
-        $http({
-          method: 'GET',
-          url: "http://127.0.0.1:8000/images/?q" + $scope.image
-          
-        }).then(function successCallback(response) {
-              console.log(response)
-          }, function errorCallback(response) {
-                alert("error");
-         });
-         console.log(url)
-
-      }
-
-});
-module.controller("RouteController1", function($scope) {
-    $scope.test="flickr photos"
+        $scope.tag = '/images/?q=' + $scope.image
+        $http.get($scope.tag).success(function(data, status, headers, config){
+        $scope.tag_data = {}
+        for(i=0; i<data.length; i++){
+            $scope.tag_data[i] = data[i];
+        }
+      }).error(function(data,status,headers,config){
```

Comment:
I think you can get rid of this `error` promise as you're not doing anything in it. just check though